doc/usecases/event_notfication.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Event Notification
+
+## Summary
+
+- **Scope:** Notifications
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user gets a notfication when another user has an event.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app and is logged in.
+  - A user triggers an event (e.g. has finished planning their map)
+  - The users are located nearby (requires location settings of both users, nearby is a defined radius).
+- **Main success scenario:**
+  The user gets a notification which informs them about the event (e.g. virtual invitation to the map).
+- **Alternative scenario:**
+  One or both of the users have no location information: The user doesn't receive a notification.
```

Comment:
No, if the sending user doesn't have a location, it must be an error. Users without location, receive all notifications (with the additional text that they get it because they don't have a location set).